src/widgets/WalletModal/ConnectModal.tsx

```diff
@@ -59,6 +57,34 @@ const getPreferredConfig = (walletConfig: Config[]) => {
   ];
 };
 
+const ScrollWrapper = styled(Box)`
+  overflow-x: hidden;
+  margin-right: 6px;
+  align-self: stretch;
+  ${({ theme }) => theme.mediaQueries.sm} {
+    margin-right: 14px;
+  }
+  &::-webkit-scrollbar {
+    width: 4px;
+  }
+  &::-webkit-scrollbar-track {
+    background-color: ${({ theme }) => theme.colors.gray200};
+    box-shadow: none;
+  }
+  &::-webkit-scrollbar-thumb {
+    background-color: ${({ theme }) => theme.colors.textSubtle};
+  }
+`;
+
+const StyledText = styled(Text)`
+  align-self: flex-start;
+`;
+
+const StyledButton = styled(Button)`
+  font-size: 16px;
+  font-weight: 500;
```

Comment:
styled system don`t have font-weight: 500
only 400 & 600
if you need bold text, use 600